src/service.js

```diff
@@ -189,6 +190,27 @@ module.exports = function(mixinOptions) {
 							return Array.isArray(dataLoaderKey)
 								? await dataLoader.loadMany(dataLoaderKey)
 								: await dataLoader.load(dataLoaderKey);
+						} else if (fileUploadArg != null && args[fileUploadArg] != null) {
+							if (Array.isArray(args[fileUploadArg])) {
+								return await Promise.all(
+									args[fileUploadArg].map(async uploadPromise => {
+										const {
+											createReadStream,
+											...$fileInfo
+										} = await uploadPromise;
+										const stream = createReadStream();
+										return await context.ctx.call(actionName, stream, {
```

Comment:
I believe the `await` here is redundant since the `Promise.all` would still catch any errors and it is being awaited.